File: CppCompiler/forloop4_1.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main()
{
int i,j,n;
double sum=0,sum1=0,k=1;
cout<<"enter no:-";
cin>>n;
for(i=1;i<=n;i+=2)
{
for(j=1;j<=i;j++)
{
if(j%2!=0)
{
cout<<j;
if(j<i)
{
cout<<"+";
}
sum=sum+j;
}
}
cout<<"/"<<k;
if(i<n)
{
cout<<" + ";
}
sum1=sum1+sum/k++;
sum=0;
}




cout<<"sum of seriese="<<sum1;
return 0;
}

File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop41.java
package sample;
import java.util.*;
class Forloop41 
{
 public static void main (String[] ar)
 {
   int i,j,n;
   double sum1=0,sum=0;
   System.out.println("enter no of term");
   Scanner m=new Scanner(System.in);
   n=m.nextInt();
   for(i=1;i<=n;i++)
   {
    System.out.print(i+"/");
    for(j=1;j<=i;j++)
    {
      System.out.print(j);
      if(j<i)
      {
        System.out.print("+");
      
      }
      sum1=sum1+j;
    }
 sum=sum+i/sum1;
 sum1=0;
 if(i<n)
 {
   System.out.print(" + ");
 }
    
   }
   System.out.print("sum of seriese="+sum); 
 }
  
}

